### ISSUE TITLE
fix: localized sitemaps include all routes on `nuxt generate`

### ISSUE THREAD

**Issue by dargmuesli** [harlan-zw/nuxt-simple-sitemap#131]
### Describe the bug

Running `nuxt generate` and `npx serve .output/public` shows for `/en-sitemap.xml`

![image](https://github.com/harlan-zw/nuxt-simple-sitemap/assets/4778485/53839013-0b60-43a6-9470-e71307c849a2)

Especially does it show `/` **and** `/en`, while only `/en` should be shown (as it does with `nuxt dev`).

---

Maybe it should also not show `false`, but that's another topic.

### Reproduction

https://stackblitz.com/edit/github-segxhh-fy9uaf?file=package.json

### System / Nuxt Info

```Shell
------------------------------
- Operating System: Linux
- Node Version:     v16.20.0
- Nuxt Version:     3.7.1
- CLI Version:      3.8.3
- Nitro Version:    2.6.3
- Package Manager:  npm@9.4.2
- Builder:          -
- User Config:      modules, i18n
- Runtime Modules:  @nuxtjs/i18n@8.0.0-rc.4, nuxt-simple-sitemap@3.2.7
- Build Modules:    -
------------------------------
```

**Comment by dargmuesli** [comment]
You're too fast! 🔥

**Comment by harlan-zw** [comment]
Thanks for catching this one :+1: Available in 3.2.8

Also I pushed a fix so that it correctly asserts a site url is set if you're prerendering (fixes the `fase`) issue.
